inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -92,7 +92,11 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*ReduceOpsLayerTest.*type=Mean_.*netPRC=U64.*)",
 
         // Issue: 62746
-        R"(smoke_CachingSupportCase_CPU/LoadNetworkCacheTestBase.CompareWithRefImpl/ReadConcatSplitAssign_f32_batch1_CPU)"
+        R"(smoke_CachingSupportCase_CPU/LoadNetworkCacheTestBase.CompareWithRefImpl/ReadConcatSplitAssign_f32_batch1_CPU)",
+        // Issue: 65862
+        R"(smoke_PrePostProcess.*convert_element_type_and_mean.*)",
+        R"(smoke_PrePostProcess.*tensor_element_type_and_mean.*)",
+        R"(smoke_PrePostProcess.*lvalues_multiple_ops.*)"
```

Comment:
Maybe we need to restore precisions to stick to ngraph::Function ones? Because CNNNetwork changes the precisions and CPU can produce different results with compare to reference implementation